### PR TITLE
AWS CLI bash script: Alex

### LINKED ISSUE #34: Create shell scripts for the weekly pipeline folder. READ DESCRIPTION
CREATE NEW ISSUE WHEN CREATING SHELL SCRIPTS.

As a DevOps engineer, I want to create Bash scripts for connecting to an AWS RDS database and for dockerizing a CI/CD pipeline, so that I can automate database access and containerize the pipeline for easier deployment and execution in a consistent environment.

Useful shell script examples:
- Connecting to database
- Initialising/reseting/seeding databse
- AWS authenticate image
- AWS tag image
- Push image to AWS
- - Build + run images

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #34 

What I have done:
- Created deploy.sh
- Implemented deploy function that utilises environment variables to:
1. Authenticate AWS ECR repo
2. Build docker image
3. Tag docker image
4. Push Docker image

### CODE REVIEW

File: dashboard/deploy.sh
Comment (CalebMathers) [inline dashboard/deploy.sh:1]: Lo and behold, this is truly splendid—a most agreeable method for automating the upload of Docker images.

File: database/connect.sh
Comment (CalebMathers) [inline database/connect.sh:1]: Truly a matter of profound intrigue, most captivating to the discerning mind!

File: database/query.sh
Comment (CalebMathers) [inline database/query.sh:1]: ‘Tis most peculiar; mayhap we shall hope this doth not besmirch our esteemed files!

**Review by CalebMathers — APPROVED** [review]

File: dashboard/deploy.sh
Comment (olliejth) [reply dashboard/deploy.sh:1]: A very good use of a bash script, this will save a lot of time later.

**Review by olliejth — APPROVED** [review]
Again, really good use of shell scripts, these will really help throughout, a good basis early on.

File: dashboard/deploy.sh
Comment (MurasatoNaoya) [reply dashboard/deploy.sh:1]: Works well for our CI/CD plan an automated image build and push to our ECR with login using local environment variables.

**Review by MurasatoNaoya — APPROVED** [review]

**Pull request merged**